vernac/comInductive.ml

```diff
@@ -29,6 +29,14 @@ open EConstr
 
 module RelDecl = Context.Rel.Declaration
 
+type flags = {
```

Comment:
Often
```ocaml
module Flags = struct
   type t = { ... }
end
```
works a bit better as to scope field names such as `poly` (c.f. Jane Street coding rules)